action.yml

```diff
@@ -56,6 +56,7 @@ runs:
         sys.exit(proc.returncode)
         "
 
+        pip install pip-requirements-parser
```

Comment:
Hm, do we test anywhere that the action as defined in the branch works and runs properly?

Reply:
And in that log I do see:
```python
Collecting pip-requirements-parser
  Downloading pip_requirements_parser-32.0.1-py3-none-any.whl (35 kB)
Requirement already satisfied: pyparsing in /usr/lib/python3/dist-packages (from pip-requirements-parser) (2.4.7)
Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from pip-requirements-parser) (23.1)
Installing collected packages: pip-requirements-parser
Successfully installed pip-requirements-parser-32.0.1
```